backend/Dev.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install system dependencies (unzip)
+RUN apt-get update && apt-get install -y unzip
+
+# Install Python dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy application code
+COPY ./app /app
+
+# Run the FastAPI app
+CMD ["uvicorn", "main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
```

Comment:
@pulgamecanica does the `--reload` do hot reloading here? I'm wonder what watchmedo is used for then?